src/quo2/components/selectors/disclaimer/view.cljs

```diff
@@ -5,12 +5,13 @@
             [react-native.core :as rn]))
 
 (defn view
-  [{:keys [on-change accessibility-label container-style]} label]
+  [{:keys [checked? on-change accessibility-label container-style]} label]
```

Comment:
I added the `checked?` property to the disclaimer because it was not able to have a controlled state and it was required since the component is remounted depending on input focus.

Reply:
how about adding a component test for this prop? 🤔